README.md

```diff
@@ -41,36 +41,20 @@
   </a>
 </p>
 
-Semgrep is a simple and flexible [command-line](#installation) and [online](https://semgrep.live/) tool for static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
-
-Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with regexes.
-
-Visit [Installation](#installation) and [Usage](#usage) to get started.
-
-## Installation
-> Want to skip installation? You can run Semgrep online via [semgrep.live](https://semgrep.live/).
-
-On macOS, binaries are available via Homebrew:
-
-```bash
-$ brew install semgrep
-```
-
-On Ubuntu, an install script is available with each [release](https://github.com/returntocorp/semgrep/releases/download/v0.10.1/semgrep-v0.10.1-ubuntu-generic.sh)
+Semgrep is a command-line tool for static analysis. Quickly write custom rules or use [pre-built checks]() to enforce code and security standards in your project. Semgrep runs locally and in CI; it works offline and never sends your code anywhere.
 
-```bash
-$ ./semgrep-v0.10.1-ubuntu-generic.sh
-```
+Quickly get started with Semgrep:
 
-To try Semgrep without installation, you can also run it via Docker:
+1. [Installation]()
+1. [Writing custom rules]()
+1. [Run pre-built rules]()
+1. [Run Semgrep continously in CI]()
 
-```
-$ docker run --rm -v "${PWD}:/home/repo" returntocorp/semgrep --help
-``` 
+## Overview
 
-See [Usage](#usage) to learn about running pre-built rules and writing custom ones.
+Semgrep combines the convenient and iterative style of `grep` with the powerful features of an Abstract Syntax Tree (AST) matcher and intra-file dataflows. Easily find function calls, class or method definitions, and more without having to understand ASTs or wrestle with a regex.
 
-## Motivation
+It began as a Facebook project called [pfff](https://github.com/facebookarchive/pfff) in 2009, where it was used to enforce nearly 1000 "paved road" rules on Facebook's codebases. Fast forward to 2019 when [r2c](https://r2c.dev/) revitalized the project after its original author, Yoann Padioleau ([@aryx](https://github.com/aryx)), joined the team. Since its revitalization Semgrep has grown to support many languages including Python, Golang, JavaScript, and Java. It's the latest in a long line of variant analysis and code automation tools with a lineage that connects to [Coccinelle](http://coccinelle.lip6.fr/).
```

Comment:
Is this important enough to be in the overview?

Reply:
Up for debate. Sounds like you feel like it doesn't make the cut?

I find it impressive that this tool has been widely used within Facebook and isn't untested tech: it works. We've been compared to Coccinelle before on Twitter, so I thought the acknowledgement was warranted here that this is derived from it.

Overall this is social proof. Happy to through another channel for doing that or saying it's uneeded.